integrations/liquid/proguard-rules.pro

```diff
@@ -27,8 +27,12 @@
 -keep class org.dash.wallet.integration.liquid.** {
     public protected private *;
 }
+-keep class org.dash.wallet.integration.coinbase_integration.** {
+    public protected private *;
+}
 -keep interface org.dash.wallet.integration.uphold.** {*;}
```

Comment:
Do we need coinbase and uphold items in the liquid module?  Looks like the previous file had uphold though this is for liquid.

Reply:
If i removed  coinbase transfer api  failed